Jenkinsfile

```diff
@@ -43,7 +54,7 @@ elifeLibrary {
                 if (isNew) {
                     sh "IMAGE_TAG=${commit} " +
                         "docker-compose -f docker-compose.yml -f docker-compose.ci.yml run " +
-                        "sciencebeam-utils twine upload dist/*"
```

Comment:
after merge I'll change this to a tag-based build

Reply:
that would have been my next ququestion